src/containers/Interfaces/OrdinalBin.js

```diff
@@ -45,7 +45,7 @@ class OrdinalBin extends Component {
             id={'NODE_BUCKET'}
             itemType="EXISTING_NODE"
             onDrop={this.onDrop}
-            sortOptions={prompt.bucketSortOrder}
+            sortOOrder={prompt.bucketSortOrder}
```

Comment:
Typo here

Reply:
Ha! I think this is literally down to the train ride being bumpy.